manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.installer.yaml

```diff
@@ -0,0 +1,45 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Waterfox.Waterfox
+PackageVersion: G6.0.3
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 7.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Protocols:
+- file
+- http
+- https
+FileExtensions:
+- htm
+- html
+- pdf
+- url
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64

```

Comment:
Waterfox doesn't require the Visual C++ redist files as they come bundled already.

Reply:
This was the result of another failure in #121246 . Did something change between 6.0.2 and 6.0.3?

Reply:
Yes, I added the missing redist files. Didn't realise they were missing 😅